pkg/kube/container/container_test.go

```diff
@@ -54,6 +65,16 @@ func TestContainer(t *testing.T) {
 	assert.Equal(t, int32(5), readinessProbe.PeriodSeconds)
 	assert.Equal(t, "exec", readinessProbe.Exec.Command[0])
 
+	liveNessProbe := c.LivenessProbe
+	assert.Equal(t, int32(15), liveNessProbe.FailureThreshold)
+	assert.Equal(t, int32(10), liveNessProbe.PeriodSeconds)
+	assert.Equal(t, "liveness-exec", liveNessProbe.Exec.Command[0])
+
+	assert.True(t, reflect.DeepEqual(c.Resources, resourcerequirements.Defaults()))
```

Comment:
I think `assert.Equal` uses `DeepEqual` under the hood, we might not need to do this manually.

Reply:
it passes using `assert.Equal` 👍